### PR TITLE
Added models and fixed namespaces

### PR DESCRIPTION
**Pull request by Kartang12**
Добавил модельки и починил неймспейсы (у нас тут инногачи, а не клин архитектура)

### CODE REVIEW

File: src/Domain/Common/PetBody.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+public class PetBody
+{
+    public NoseTypeEnum Nose { get; set; }
+    public EyesTypeEnum Eyes { get; set; }
+    public MouthTypeEnum Mouth { get; set; }
+    public BodyTypeEnum Body { get; set; }
+    public virtual User User { get; set; }
+}
+
+public enum NoseTypeEnum
```
Comment (sdechko1) [inline src/Domain/Common/PetBody.cs:11]: I think you should rename this to `NoseType` so it does not contain Enum

**Review by sdechko1 — APPROVED** [review]

File: src/Domain/Common/PetType.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace InnoGotchi.Domain.Common;
+
+public class PetType: BaseEntity
+{
+    public string Name { get; set; }
+}
```
Comment (klimenkatechart) [inline src/Domain/Common/PetType.cs:6]: Consider deleting this Class, I think it's a good idea for real-life project, but in your scenario we don't want to overcomplicated things

**Review by klimenkatechart — CHANGES_REQUESTED** [review]

File: src/Domain/Common/Pet.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿namespace InnoGotchi.Domain.Common;
+
+public class Pet: BaseEntity
+{
+    public string Name { get; set; }
+    public PetType Type { get; set; }
+    public PetBody Body{ get; set; }
+    public int Age { get; set; }
+    public int Hunger { get; set; } // 0-3
```
Comment (dakamakat) [inline src/Domain/Common/Pet.cs:9]: Consider to use actual enums for these statuses avoid magic numbers in code

File: src/Domain/Common/PetBody.cs (hunk 1 above)
Comment (Kartang12) [reply src/Domain/Common/PetBody.cs]: Done

File: src/Domain/Common/User.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace InnoGotchi.Domain.Common;
+
+public class User
+{
+    public Guid Id { get; set; }
```
[the remark this answers is not in the document]
Comment (Kartang12) [reply src/Domain/Common/User.cs]: Done

File: src/Domain/Common/Pet.cs (hunk 3 above)
Comment (Kartang12) [reply src/Domain/Common/Pet.cs]: true

**Pull request merged**